exercises/allergies/src/Allergies.hs

```diff
@@ -14,4 +14,4 @@ allergies :: Int -> [Allergen]
 allergies = error "You need to implement this function."
```

Comment:
This function also needs an explicit argument. You can call it `n` to keep consistency.